system/vrops-custom-importer/values.yaml

```diff
@@ -0,0 +1,22 @@
+owner-info:
+  support-group: compute
+  maintainers:
+    - Christoph Richter
+  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/system/vrops-custom-importer
+namespace: FROM-EXTERNAL
+schedule: FROM-EXTERNAL
+image:
+  registry: FROM-EXTERNAL
+  name: FROM-EXTERNAL
+vrops:
+  cluster: FROM-EXTERNAL
+  username: FROM-EXTERNAL
+  password: FROM-EXTERNAL
+vc:
+  cluster: FROM-EXTERNAL
+  username: FROM-EXTERNAL
+  password: FROM-EXTERNAL
+authSource: FROM-EXTERNAL
+debug: FROM-EXTERNAL
+insecure: FROM-EXTERNAL
+region: FROM-EXTERNAL
```

Comment:
all `FROM-EXTERNAL` should be empty. Add a ci folder and specify some values to get it through linting.